colcon_devtools/verb/version_check.py

```diff
@@ -31,16 +33,38 @@ def main(self, *, context):  # noqa: D102
         for entry_point in colcon_extension_points.values():
             distributions.add(entry_point.dist)
 
-        pypi = ServerProxy('https://pypi.python.org/pypi')
+        base_url = 'https://pypi.python.org/pypi/{project}/json'
         for dist in sorted(distributions, key=lambda d: d.project_name):
-            versions = pypi.package_releases(dist.project_name)
-            if not versions:
+            url = base_url.format(project=dist.project_name)
+
+            try:
+                response = urllib.request.urlopen(url)
+            except urllib.error.HTTPError:
```

Comment:
I think if a package was not found an `HTTPError` is raised with status 404. In that case the resulting error message is incorrect.

Reply:
I fixed this in 997b685